bin/zeppelin-daemon.sh

```diff
@@ -67,12 +67,17 @@ if [[ -d "${ZEPPELIN_HOME}/zeppelin-server/target/classes" ]]; then
   ZEPPELIN_CLASSPATH+=":${ZEPPELIN_HOME}/zeppelin-server/target/classes"
 fi
 
-addJarInDir "${ZEPPELIN_HOME}"
-addJarInDir "${ZEPPELIN_HOME}/lib"
-addJarInDir "${ZEPPELIN_HOME}/zeppelin-interpreter/target/lib"
-addJarInDir "${ZEPPELIN_HOME}/zeppelin-zengine/target/lib"
-addJarInDir "${ZEPPELIN_HOME}/zeppelin-server/target/lib"
-addJarInDir "${ZEPPELIN_HOME}/zeppelin-web/target/lib"
+addEachJarInDir "${ZEPPELIN_HOME}"
+addEachJarInDir "${ZEPPELIN_HOME}/lib"
+addEachJarInDir "${ZEPPELIN_HOME}/zeppelin-interpreter/target/lib"
+addEachJarInDir "${ZEPPELIN_HOME}/zeppelin-zengine/target/lib"
+addEachJarInDir "${ZEPPELIN_HOME}/zeppelin-server/target/lib"
+addEachJarInDir "${ZEPPELIN_HOME}/zeppelin-web/target/lib"
```

Comment:
Yes.

The current code generates CLASSPATH in form

directory1/*:directory2/*

The above code generates:

directory1/jar1:directory1/jar2:directory2/jar3

I'm not sure if the "*" was a shortening approach ...